src/commands/images/runImage.ts

```diff
@@ -26,7 +27,7 @@ async function runImageCore(context: IActionContext, node: ImageTreeItem | undef
     }
 
     const image = await node.getImage();
-    const inspectInfo = await image.inspect();
+    const inspectInfo = await callDockerode(async () => image.inspect());
```

Comment:
Why do we need callDockerrode now?

Reply:
This spot should've had it prior to now. Karol's context changes make flowing through `callDockerode` preferable, because it handles checking for context switches.